### PR TITLE
fix: forgotten labels

### PR DESCRIPTION
**Pull request by Berkmann18**
There are no direct changes in the behaviour other than the fact that the bot fills in the gaps where one of the following isn't true:
- no reviews
- one review requesting changes
- all reviews were accepted

Which seems to be what steamed the issue in #119 where some labels (namely: "PR: unreviewed") were forgotten.

There are other minor changes that include a small loop fusion, typo fix and addition of a detail in an error report regarding statuses.
There are more things that could be improved but I decided to leave those for now.

### CODE REVIEW

**Comment by sotayamashita** [comment]
@Berkmann18 Thank you for creating the PR :+1: I will review this on this weekend.

File: test/fixtures/payload.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "merged": {
         "data": {
           "title": "Pull Request with unreviewed and merged",
-          "labels": [{ "name": "PR: unreviewed" }],
+          "labels": [{ "name": "Status: Review Needed :passport_control:" }],
```
Comment (TiagoDanin) [inline test/fixtures/payload.json]: Label `Status: Review Needed :passport_control:` ?

Comment (Berkmann18) [reply test/fixtures/payload.json]: I needed to have consistent labels with my set but it's been reverted.

Comment (sotayamashita) [reply test/fixtures/payload.json]: @TiagoDanin Thank you for your help. I would appreciate if you also review this PR.

**Comment by sotayamashita** [comment]
I'm sorry to have kept you waiting for a long time. I will look into it ASAP!

**Comment by Berkmann18** [comment]
@sotayamashita Okay, no problem.

File: test/fixtures/payload.json (hunk 1 above)
Comment (TiagoDanin) [reply test/fixtures/payload.json]: @Berkmann18 But this is part of another test: "PR: merged"
"PR: merged" Test:
- Load playload "merged"
- Event received: "merged"
- Check Labels
- Remove labels of PR-triage ("PR: unreviewed" Not found... and label "Status: Review Needed :passport_control:" is not part of PR-triage)
- Add label "PR: merged"
- Done!

Comment (Berkmann18) [reply test/fixtures/payload.json]: @TiagoDanin Again, I reverted that, look at the latest commit.

**Comment by Berkmann18** [comment]
@sotayamashita Why did you close this?

**Pull request closed without merging**